package.json

```diff
@@ -50,7 +49,7 @@
     "snazzy": "^9.0.0",
     "split2": "^4.0.0",
     "standard": "^17.0.0",
-    "tap": "^16.3.0",
+    "tap": "16.3.0",
```

Comment:
Is this meant to be pinned?

Reply:
YEs it is, there is an issue with the 16.3.1 version due a babel dependency that breaks the `npm install` on windows/node 14

https://github.com/tapjs/node-tap/issues/843

Reply:
gotcha!